package.json

```diff
@@ -46,16 +49,16 @@
     "husky": "^7.0.2",
     "lint-staged": "^12.1.2",
     "node-plantuml": "windranger-io/node-plantuml#fcfc730",
-    "bitdao-token-contract": "windranger-io/bit-token-contract#v1.0.0",
     "prettier": "^2.4.1",
+    "prettier-plugin-solidity": "^1.0.0-beta.18",
     "ts-node": "^10.2.1",
     "typechain": "^5.1.2",
     "typescript": "^4.4.2",
-    "web3": "^1.6.1",
-    "prettier-plugin-solidity": "^1.0.0-beta.18"
+    "web3": "^1.6.1"
   },
   "dependencies": {
-    "@openzeppelin/contracts-upgradeable": "^4.3.3",
```

Comment:
suggestion: move the OZ dependencies to dev

Strictly speaking once the contract are compile, there's no dependency i.e. the compiled contract can be deployed and run successfully without the OZ dependencies.